File: Python/tests/test_ts_par4.py
import pytest

from hbayesdm.models import ts_par4


def test_ts_par4():
    _ = ts_par4(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_ts_par4.R
context("Test ts_par4")
library(hBayesDM)

test_that("Test ts_par4", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(ts_par4(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
